css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,67 @@
+.container {
+  font-family: 'DM Sans', sans-serif;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.col-fifth {
+  width: calc(100%/6);
+  margin: 0 20px;
+}
+
+.item-picture-black-dress {
+  background-image: url('images/dress1.jpeg');
+  background-size: contain;
```

Comment:
Everything in each of these rules for the images is repeated, except for the `background-image` property. You would benefit from creating one class with all of the repeated properties that you can apply to each image, and have individual rules for each `background-image` as well. It ends up being more CSS rules, but less repetition!